Lib/MigrationVersion.php

```diff
@@ -373,8 +373,11 @@ public function run($options) {
 				$migration->info = $info;
 
 				try {
+					$hostName = gethostname();
+					echo __d('migrations', "Migration starting on $hostName: ". date('Y-m-d H:i:s') . "\n");
```

Comment:
@markvince - Non blocking suggestion. What if you switched the `date` format to either `c` for an ISO 8601 date or add one of the other timezone offset options, eg. O or P. I just think having the timezone offset will be helpful in debugging.

Reply:
Ok.  Changed format to ('c').